tests/e2e/csi_snapshot_utils.go

```diff
@@ -667,7 +669,7 @@ func createPVCAndQueryVolumeInCNS(client clientset.Interface, namespace string,
 
 	ginkgo.By("Expect claim to provision volume successfully")
 	persistentvolumes, err := fpv.WaitForPVClaimBoundPhase(client,
-		[]*v1.PersistentVolumeClaim{pvclaim}, framework.ClaimProvisionTimeout)
+		[]*v1.PersistentVolumeClaim{pvclaim}, framework.ClaimProvisionTimeout*2)
```

Comment:
please add a comment why wait time of 2 * Provisiontimeout

Reply:
This is required if any vcenter service is down, then PVC takes time to come up